src/app/Http/Requests/ValidateEventIndexRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LaravelEnso\Calendar\app\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use LaravelEnso\Calendar\app\Facades\Calendars;
+
+class ValidateEventIndexRequest extends FormRequest
```

Comment:
either here or in the Controller where we're using the request we should, again, ensure that the user has access to all the calendars he's fetching events for.